zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/DocumentController.java

```diff
@@ -44,7 +45,8 @@ public class DocumentController {
   private final ObjectMapper objectMapper;
 
   public DocumentController(
-      final DocumentServices documentServices, final ObjectMapper objectMapper) {
+      final DocumentServices documentServices,
+      @Qualifier("gatewayRestObjectMapper") final ObjectMapper objectMapper) {
```

Comment:
Hello @tmetzke 

I have applied the suggestions, The introduction of a bean for _**yamlMapper**_ , introduced the Bean conflict over here and on the imported library  **MappingJackson2HttpMessageConverter**

In order to solve it, I add the respective qualifiers and redeclared the bean for MappingJackson2HttpMessageConverter.